docs/ref/models/instances.txt

```diff
@@ -533,7 +533,7 @@ value, the field will be added to the updated fields.
 Deleting objects
 ================
 
-.. method:: Model.delete([using=DEFAULT_DB_ALIAS])
+.. method:: Model.delete([using=DEFAULT_DB_ALIAS, keep_parents=False])
```

Comment:
should have a `.. versionchanged:: 1.9` annotation at the bottom of this method indicating that the `keep_parents` parameter was added.